### PR TITLE
Apr 2065 outcome comments

### PR DESCRIPTION
**Pull request by chrisfoster76**

### CODE REVIEW

**Review by martin-lush — APPROVED** [review]

**Review by MarkFCain — APPROVED** [review]

File: src/SFA.DAS.RoatpGateway.Web/Validators/RoatpGatewayApplicationViewModelValidator.cs [hunk 1]
```diff
@@ -85,8 +99,21 @@ public async Task<ValidationResponse> Validate(RoatpGatewayApplicationViewModel
...
+                    {
+                        validationResponse.Errors.Add(new ValidationErrorDetail("OptionExternalRejectedText", ErrorEnterExternalComments));
+                    }
+                    else
+                    {
+                        var wordCount = viewModel.OptionExternalRejectedText.Split(new[] { " " }, StringSplitOptions.RemoveEmptyEntries).Length;
+                        if (wordCount > 150)
+                        {
+                            validationResponse.Errors.Add(new ValidationErrorDetail("OptionExternalRejectedText", TooManyWords));
+                        }
```
Comment (martin-lush) [inline src/SFA.DAS.RoatpGateway.Web/Validators/RoatpGatewayApplicationViewModelValidator.cs:113]: Does this tally up with the view? I think I saw 500 words there

File: src/SFA.DAS.RoatpGateway.Web/Views/Gateway/ConfirmOutcome.cshtml [hunk 2]
```diff
@@ -197,10 +199,31 @@
                                           id="OptionFailedText"
                                           name="OptionFailedText"
                                           rows="5"
-                                          aria-describedby="OptionFailedText-info">@Model.OptionFailedText</textarea>
+                                          aria-describedby="OptionFailedText-info OptionFailedText--hint">@Model.OptionFailedText</textarea>
                                 <span id="OptionFailedText-info" class="govuk-hint govuk-character-count__message" aria-live="polite"></span>
                             </div>
                         </div>
+                        <div class="govuk-character-count" data-module="govuk-character-count" data-maxwords="500">
```
Comment (martin-lush) [inline src/SFA.DAS.RoatpGateway.Web/Views/Gateway/ConfirmOutcome.cshtml:206]: Probably should be 150 words here

File: src/SFA.DAS.RoatpGateway.Web/Views/Gateway/ConfirmOutcome.cshtml [hunk 3]
```diff
@@ -232,10 +256,31 @@
                                           id="OptionRejectedText"
                                           name="OptionRejectedText"
                                           rows="5"
-                                          aria-describedby="OptionFailedText-info">@Model.OptionRejectedText</textarea>
+                                          aria-describedby="OptionRejectedText-info OptionRejectedText-hint">@Model.OptionRejectedText</textarea>
                                 <span id="OptionRejectedText-info" class="govuk-hint govuk-character-count__message" aria-live="polite"></span>
                             </div>
                         </div>
+                        <div class="govuk-character-count" data-module="govuk-character-count" data-maxwords="500">
```
Comment (martin-lush) [inline src/SFA.DAS.RoatpGateway.Web/Views/Gateway/ConfirmOutcome.cshtml:263]: 150 words here?

File: src/SFA.DAS.RoatpGateway.Web/Views/Gateway/ConfirmOutcome.cshtml (hunk 2 above)
Comment (gdbarnes) [reply src/SFA.DAS.RoatpGateway.Web/Views/Gateway/ConfirmOutcome.cshtml:206]: That's Fail - External comments right? AC3 and the content doc both say 500 words.

File: src/SFA.DAS.RoatpGateway.Web/Validators/RoatpGatewayApplicationViewModelValidator.cs (hunk 1 above)
Comment (gdbarnes) [reply src/SFA.DAS.RoatpGateway.Web/Validators/RoatpGatewayApplicationViewModelValidator.cs:113]: Yep.. I think you're right here @martin-lush. All the external comments seem to have a 500 word limit, though this is really for @chrisfoster76 to update.

File: src/SFA.DAS.RoatpGateway.Web/Views/Gateway/ConfirmOutcome.cshtml (hunk 3 above)
Comment (gdbarnes) [reply src/SFA.DAS.RoatpGateway.Web/Views/Gateway/ConfirmOutcome.cshtml:263]: Rejected -> External comments. AC5 and the content doc both say 500 words.

**Review by martin-lush — APPROVED** [review]

**Pull request merged**